source/apickli/apickli-gherkin.js

```diff
@@ -25,38 +26,38 @@ const callbackWithAssertion = function(callback, assertion) {
     }
 };
 
-module.exports = function() {
-    this.registerHandler('BeforeScenario', function(scenario, callback) {
-        stepContext.scenario = scenario.getName;
-        callback();
-    });
+defineSupportCode(function({Before}) {
+    Before(function(scenarioResult, callback) {
+        // https://github.com/cucumber/cucumber-js/issues/891
+        // stepContext.step = step.getName;
+        // stepContext.scenario = scenario.getName;
 
-    this.registerHandler('BeforeStep', function(step, callback) {
-        stepContext.step = step.getName;
         callback();
     });
+});
 
-    this.Given(/^I set (.*) header to (.*)$/, function(headerName, headerValue, callback) {
+defineSupportCode(function({Given, When, Then}) {
```

Comment:
If you are talking about the name `defineSupportCode`, then it is not our code/choice. That handler is defined by cucumber 3 api: https://github.com/cucumber/cucumber-js/blob/master/docs/support_files/step_definitions.md